auction9-frontend/src/app/data-table/data-table-details/data-table-details.component.html

```diff
@@ -23,22 +25,31 @@ <h4>Status:
       </ng-template>
   	</h4>
 	<div class="bid-number-container">
-		<h4>Number of bids: 
-			<button mat-mini-fab color="primary" matTooltip="Go to history of all bids"
-        			aria-label="Button that will redirect user to the history of all bids for this product">
+		<h4>Number of bids:
+      <button
+        mat-mini-fab
+        (click)="showHistory()"
+        color="primary"
+        matTooltip="Go to history of all bids"
+        aria-label="Button that will redirect user to the history of all bids for this product">
 		      +11
-		    </button>
+      </button>
 		</h4>
-	    
+
   	</div>
     <p>
       {{auction?.description}}
     </p>
   </mat-card-content>
   <mat-card-actions>
   	<mat-form-field class="input-bid">
-	    <input matInput placeholder="Offer must start from: {{auction?.price + 1}} din" value="{{auction?.price + 1}}" min="{{auction?.price + 1}}" required>
+      <input
+        matInput
+        placeholder="Offer must start from: {{auction?.price + 1}} din"
```

Comment:
We should make the inncrement configurable. I'll create a ticket

Reply:
Stefan said he will take care of that, okay.